Library/Homebrew/dev-cmd/install-bundler-gems.rb

```diff
@@ -14,13 +14,18 @@ def install_bundler_gems_args
     Homebrew::CLI::Parser.new do
       description <<~EOS
         Install Homebrew's Bundler gems.
+
+        Not (yet) working on Apple Silicon.
       EOS
 
       named_args :none
     end
   end
 
   def install_bundler_gems
+    # TODO: update description above if removing this.
+    raise UsageError, "not (yet) working on Apple Silicon!" if Hardware::CPU.arm?
```

Comment:
Maybe we can allow this command but block the commands we know don't work? Using the system ruby, `install-bundler-gems` does install gems, it's just that some don't work.